pandas/core/computation/eval.py

```diff
@@ -206,19 +205,33 @@ def eval(expr, parser='pandas', engine=None, truediv=True,
         The number of prior stack frames to traverse and add to the current
         scope. Most users will **not** need to change this parameter.
     target : a target object for assignment, optional, default is None
-        essentially this is a passed in resolver
-    inplace : bool, default True
-        If expression mutates, whether to modify object inplace or return
-        copy with mutation.
-
-        WARNING: inplace=None currently falls back to to True, but
-        in a future version, will default to False.  Use inplace=True
-        explicitly rather than relying on the default.
+        This is essentially passed into the resolver and is used when there
+        is variable assignment in the expression. If so, then `target` must
+        support item assignment with string keys, and if a copy is being
+        returned, it must also support `.copy()`.
+    inplace : bool, default False
+        If `target` is provided, and the expression mutates `target`, whether
+        to modify `target` inplace. Otherwise, return a copy of `target` with
+        the mutation.
 
     Returns
     -------
     ndarray, numeric scalar, DataFrame, Series
 
+    Raises
+    ------
+    ValueError : There are many instances where such an error can be raised:
```

Comment:
Can you put the "There are many instances where such an error can be raised:" also on the next line? (and then the colon `:` is not needed)